codegen.ts

```diff
@@ -0,0 +1,27 @@
+import { ContractsENV, SUBGRAPH_URLS } from '@fluencelabs/deal-ts-clients'
+import type { CodegenConfig } from '@graphql-codegen/cli'
+import * as dotenv from 'dotenv'
+dotenv.config()
+
+const FLUENCE_CLIENT_NETWORK = (process.env.VITE_FLUENCE_CLIENT_NETWORK ||
+  'local') as ContractsENV
+
+const config: CodegenConfig = {
+  schema: SUBGRAPH_URLS[FLUENCE_CLIENT_NETWORK],
+  documents: 'src/**/*.graphql',
+  generates: {
+    'generated/graphql.ts': {
```

Comment:
I don't know if absolutely required but for some reason there is some configuration in deal-ts-clients that you don't have here

https://github.com/fluencelabs/compute-marketplace/blob/030e1e0b5783df444b6ea83917e3e8f81cbbd556/ts-client/codegen-config.ts#L20

maybe you should have it here as well

Reply:
I've update config here